app/controllers/comments_controller.rb

```diff
@@ -17,6 +19,7 @@ def new
 
   # GET /comments/1/edit
   def edit
+    authorize @comment
```

Comment:
We can remove this now since we already call it everywhere with the `before_action`